Grammar/python.gram

```diff
@@ -646,7 +646,7 @@ invalid_assignment:
         RAISE_SYNTAX_ERROR_KNOWN_LOCATION(a, "only single target (not tuple) can be annotated") }
     | a=expression ':' expression ['=' annotated_rhs] {
         RAISE_SYNTAX_ERROR_KNOWN_LOCATION(a, "illegal target for annotation") }
-    | a=star_expressions '=' (yield_expr | star_expressions) {
+    | a=(z=(star_expressions) '=' {z})+ (yield_expr | star_expressions) {
```

Comment:
Hm, `z=(star_expressions)` has redundant parentheses. Probably the `)` should be after `'='`?

Reply:
Yep, I have also pushed a new commit to handle mixed cases like `x = y = z = True = 3` in 06ecf9d